app/assets/javascripts/app/components/button/button.js

```diff
@@ -27,7 +27,10 @@
   function councilRaisedButton() {
     return {
       restrict: 'E',
-      scope: {},
+      scope: {
+        disabled: '=',
+        click: '&'
+      },
```

Comment:
I know, I said just to keep the consistency but in this case if you think it's ok then :+1: